pkgs/development/python-modules/pacup/default.nix

```diff
@@ -0,0 +1,37 @@
+{
+  lib,
+  poetry2nix,
+  python,
+  fetchFromGitHub,
+}:
+poetry2nix.mkPoetryApplication rec {
+  pname = "pacup";
+  version = "1.1.0";
+
+  src = fetchFromGitHub {
+    owner = "pacstall";
+    repo = pname;
+    rev = version;
+    hash = "sha256-Hl/Gq/cZz4RGYKTuyDlrhATAUYEzKEuYIm0JdToN/ZY=";
+  };
+
+  projectDir = src;
+
+  # Propagating dependencies leads to issues downstream
+  # We've already patched poetry to prefer "vendored" dependencies
+  postFixup = ''
+    rm -r $out/nix-support
+  '';
+
+  # Fails because of impurities (network, git etc etc)
+  doCheck = false;
+
+  meta = with lib; {
+    description = "Help maintainers update pacscripts";
+    longDescription = "Pacup (Pacscript Updater) is a maintainer helper tool to help maintainers update their pacscripts. It semi-automates the tedious task of updating pacscripts, and aims to make it a fun process for the maintainer!";
+    homepage = "https://github.com/pacstall/pacup";
+    license = licenses.gpl3Only;
```

Comment:
also it seems like pacup is gplv3+ according to https://github.com/pacstall/pacup/blob/master/LICENSE_BOILERPLATE